docs/OpenCLCOnVulkan.md

```diff
@@ -630,8 +633,10 @@ The OpenCL C work-item functions map to Vulkan SPIR-V as follows:
   `NumWorkgroups`.
 - `get_group_id()` is mapped to a SPIR-V variable decorated with
   `WorkgroupId`.
-- `get_global_offset()` is mapped to the `global_offset` push constant when
-  `-global-offset` is enabled, otherwise it always returns 0.
+- `get_global_offset()` is mapped to the `global_offset` spec constant or push
+  constant when `-global-offset` is enabled, otherwise it always returns 0.
```

Comment:
I'll clarify about the options, but since this is talking about how specific functions are mapped it doesn't need to be mentioned that it only occurs if used.